components/courses/app/controllers/admin/courses/mentors_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Admin
+  module Courses
+    class MentorsController < BaseController
+      helper_method :mentors
+      add_breadcrumb 'seasons.plural', :admin_courses_seasons_path
+      add_breadcrumb 'mentors.plural', :admin_courses_seasons_path
```

Comment:
you also need a breadcrumb of the season:
admin / courses / name_of_the_course / mentors